### PR TITLE
Sandbox -> development

### PR DESCRIPTION
**Pull request by jaro-m**
Changes to `constant.py` fix issues with some metrics not being accessible via APIv1.
I'm not sure about other changes, especially in `host.json`

### CODE REVIEW

File: .vs/VSWorkspaceState.json
Comment (beawitcht) [inline .vs/VSWorkspaceState.json:1]: Is this intended to be here?

Comment (jaro-m) [reply .vs/VSWorkspaceState.json:1]: The file was added here: https://github.com/publichealthengland/coronavirus-dashboard-api-v1/commit/b782536175319936f9336070299c804ee8904db0
@adebayoolabintan can explain the intension of adding the file, but I'm sure it won't affect how the service runs.

**Review by beawitcht — APPROVED** [review]

File: api_v1/api_handler/api_variables.json [hunk 1]
```diff
@@ -1 +1 @@
...ccinationDate":"int","weeklyPeopleVaccinatedSecondDoseByVaccinationDate":"int","cumPeopleVaccinatedFirstDoseByVaccinationDate":"int","cumPeopleVaccinatedSecondDoseByVaccinationDate":"int","newDailyNsoDeathsByDeathDate":"int","cumDailyNsoDeathsByDeathDate":"int","newWeeklyNsoDeathsByRegDate":"int","cumWeeklyNsoDeathsByRegDate":"int","cumWeeklyNsoDeathsByRegDateRate":"float","newCasesPCROnlyBySpecimenDateRollingSum":"int","newCasesLFDOnlyBySpecimenDateRollingRate":"float","newCasesLFDOnlyBySpecimenDate":"int","newCasesLFDConfirmedPCRBySpecimenDate":"int","newCasesLFDConfirmedPCRBySpecimenDateRollingRate":"float","cumCasesPCROnlyBySpecimenDate":"int","newCasesPCROnlyBySpecimenDateRollingRate":"float","newCasesLFDOnlyBySpecimenDateRollingSum":"int","cumCasesLFDConfirmedPCRBySpecimenDate":"int","cumCasesLFDOnlyBySpecimenDate":"int","newCasesPCROnlyBySpecimenDate":"int","newCasesLFDConfirmedPCRBySpecimenDateRollingSum":"int","cumVaccinationFirstDoseUptakeByPublishDatePercentage":"float","cumVaccinationSecondDoseUptakeByPublishDatePercentage":"float","cumVaccinationCompleteCoverageByPublishDatePercentage":"float","newVaccinesGivenByPublishDate":"int","cumVaccinesGivenByPublishDate":"int"}
```
Comment (jaro-m) [inline api_v1/api_handler/api_variables.json:1]: I'm sure this file is not used at all. I added the new metrics to it, but the file will probably be deleted soon.

File: host.json [hunk 2]
```diff
@@ -38,59 +38,63 @@
...
-            "tempFolder": null,
-            "thresholdForSnapshotting": 1,
-            "uploaderProxy": null
-        }
+  "logging": {
+    "fileLoggingMode": "debugOnly",
+    "logLevel": {
+      "default": "Trace",
+      "Host.Results": "Trace",
+      "Function": "Trace",
```
Comment (jaro-m) [inline host.json:46]: @adebayoolabintan do we want these changes in `development` environment?

Comment (adebayoolabintan) [reply host.json:46]: No. That does not need to go to daisy or prod. It used it to increase the logging level

**Comment by jaro-m** [comment]
Obsolete as this PR has been merged: https://github.com/publichealthengland/coronavirus-dashboard-api-v1/pull/109

**Pull request closed without merging**